lib/ui/myplot/new_my_plot_child_item.dart

```diff
@@ -0,0 +1,102 @@
+
+    return Column(
+      children: <Widget>[
+        Container(
+          padding: myPlotStyle.listPadding,
+          child: Row(
+            mainAxisAlignment: myPlotStyle.mainAxisAlignmentSpaceBeetwen,
+            children: <Widget>[
+              Column(
+                crossAxisAlignment: myPlotStyle.crossAxisAlignmentStart,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Text(
+                          cropsData.name ?? Strings.defaultCropNameText,
+                          style: myPlotStyle.cropNameTextStyle),
+                      Padding(
+                        padding: myPlotStyle.paddingBetweenElements,
+                      )
+                    ],
+                  ),
+                  Row(
+                    children: <Widget>[
+                    Text(currentStage, style: myPlotStyle.cropStatusTextStyle),
+                    Padding(
+                      padding: myPlotStyle.paddingBeforeCropDayCount,
+                    )
+                  ],
+                  ),
+                  Row(
+                    children: <Widget>[
+                    Container(
+                      padding: myPlotStyle.paddingForDayCount,
+                      decoration: BoxDecoration(
+                        color: myPlotStyle.primaryGreen.withOpacity(myPlotStyle.oppacityForDayCount),
+                        borderRadius: myPlotStyle.ovalRadiousForDayCount),
+                      child: Row(
+                        mainAxisSize: myPlotStyle.mainAxisSizeMin,
+                       children: <Widget>[
+                         Flexible(
+                           child: Container(
+                             child: Text(
+                              DayText,
+                              style: myPlotStyle.cropDayTextStyle,
+                           ),
+                         )
+                         )],
+                      ),
+                    )
+                  ],
+                  ),
+                ],
+              ),
+              Column(
+                children: <Widget>[
+                  _buildClipOval(cropsData, myPlotStyle),
+                ],
+              )
+            ],
+          ),
+        ),
+        Container(
+            child: buildListSeparator(myPlotStyle)
+        ),
+      ],
+    );
+  }
+
+  ClipOval _buildClipOval(CropEntity cropsData, HomeMyPlotPageStyle myPlotStyle) {
+    return ClipOval(
+                  child: NetworkImageFromFuture(
+                      cropsData.imageUrl,
+                      height: myPlotStyle.sizeForDayCountShape,
+                      width: myPlotStyle.sizeForDayCountShape,
+                      fit: BoxFit.cover
+                  ),
+                );
+  }
+
+  Widget buildListSeparator(HomeMyPlotPageStyle myPlotStyle) {
+    return Container(
+        height: myPlotStyle.separatorHeight,
+        color: myPlotStyle.separatorWhite,
```

Comment:
this has the colour name in it, If I change it to blue will this still make sense? "separatorColor" maybe?